packages/orchestration/src/orchestration-api.ts

```diff
@@ -145,17 +145,17 @@ export interface OrchestrationAccountI {
   /**
    * Transfer an amount to another account, typically on another chain.
    * The promise settles when the transfer is complete.
-   * @param amount - the amount to transfer.
-   * @param destination - the account to transfer the amount to.
-   * @param memo - an optional memo to include with the transfer, which could drive custom PFM behavior
+   * @param {AmountArg} amount - the amount to transfer.
+   * @param {ChainAddress} destination - the account to transfer the amount to.
+   * @param {IBCMsgTransferOptions} [opts] - an optional memo to include with the transfer, which could drive custom PFM behavior, and timeout parameters
    * @returns void
    *
    * TODO document the mapping from the address to the destination chain.
    */
   transfer: (
     amount: AmountArg,
     destination: ChainAddress,
-    memo?: string,
+    opts?: IBCMsgTransferOptions,
```

Comment:
@dtribble wants to review changes to `orchestration-api`. These look to me like fixups of the existing API so I wouldn't block awaitinghis review